docker-compose.yml

```diff
@@ -10,6 +10,10 @@ services:
     ports:
      - "8000:8000"
      - "1337:1337"
+     - "4444:4444"
+     - "35729:35729"
+     - "5858:5858"
```

Comment:
Is it possible to have comments in YAML? I think some indication about what each port stands for would be nice.